src/components/Interpretations/InterpretationsUnit/InterpretationList.js

```diff
@@ -59,8 +59,10 @@ export const InterpretationList = ({
                                         key={interpretation.id}
                                         interpretation={interpretation}
                                         currentUser={currentUser}
-                                        onClick={onInterpretationClick}
-                                        refresh={refresh}
+                                        reply={onInterpretationClick}
```

Comment:
Perhaps `reply` is not the most correct term, the click opens the modal where you it's possible to reply, but is not necessarily what will happen, ie. you can edit the interpretation/comment, simply read the comments or do nothing.